cpp/tests/geometry/PointCloud.cpp

```diff
@@ -808,6 +808,21 @@ TEST(PointCloud, UniformDownSample) {
                                     }));
 }
 
+TEST(PointCloud, GetFarthestPointSample) {
+    geometry::PointCloud pcd({{0, 0, 0},
```

Comment:
I would add some fraction numbers to break the symmetry. e.g. 1.5, 2.0, 5.0, ...